codec/encoder/core/src/ratectl.cpp

```diff
@@ -548,11 +571,17 @@ void RcDecideTargetBits (sWelsEncCtx* pEncCtx) {
   SRCTemporal* pTOverRc         = &pWelsSvcRc->pTemporalOverRc[pEncCtx->uiTemporalId];
 
   pWelsSvcRc->iCurrentBitsLevel = BITS_NORMAL;
+  const bool fix_rc_overshoot = pEncCtx->pSvcParam->bFixRCOverShoot;
   //allocate bits
   if (pEncCtx->eSliceType == I_SLICE) {
-    pWelsSvcRc->iTargetBits = pWelsSvcRc->iBitsPerFrame * IDR_BITRATE_RATIO;
+    if( pWelsSvcRc->iIdrNum != 0 ){
+      pWelsSvcRc->iTargetBits = pWelsSvcRc->iBitsPerFrame * pEncCtx->pSvcParam->iIdrBitrateRatio / 100;
```

Comment:
By default "sParam.iIdrBitrateRatio = IDR_BITRATE_RATIO;"
then here it is to be divided by 100? Is it wrong or the original value should be 100 * it? Otherwise, the non-first IDR will be allocated with too few bits, compared with that of the first IDR.

Reply:
Hi @huili2,
Thank you for pointing this out. You are right. The default value of iIdrBitrateRatio should be 100* IDR_BITRATE_RATIO. I have fixed this.